lib/suspenders/app_builder.rb

```diff
@@ -189,10 +169,9 @@ def setup_asset_host
         "config.assets.version = '1.0'",
         'config.assets.version = (ENV["ASSETS_VERSION"] || "1.0")'
 
-      inject_into_file(
-        "config/environments/production.rb",
-        '  config.static_cache_control = "public, max-age=#{1.year.to_i}"',
-        after: serve_static_files_line
+      configure_environment(
+        "production",
+        'config.static_cache_control = "public, max-age=#{1.year.to_i}"',
```

Comment:
> How about:
> 
> %{config.static_cache_control = "public, max-age=#{1.year.to_i}"}

That runs the interpolation while generating the app, producing: `undefined method 'year' for 1:Fixnum (NoMethodError)`.